### PR TITLE
Release/v1.6.2

### PR DESCRIPTION
**Pull request by delcroip**

### CODE REVIEW

**Comment by delcroip** [comment]
a bitt odd to see so much change between release and dev

File: IMIS/App_GlobalResources/Resource.en-cm.resx [hunk 1]
```diff
@@ -5037,7 +5037,7 @@ adults/children)</value>
 		<value>Payment Type</value>
 	</data>
 	<data name="R_RENEWALUPLOAD" xml:space="preserve">
-		<value>Renewal Upload</value>
+		<value></value>
```
Comment (hirensoni913) [inline IMIS/App_GlobalResources/Resource.en-cm.resx:5040]: Why those translations are removed?

File: IMIS/App_GlobalResources/Resource.fr-CM.resx [hunk 2]
```diff
@@ -5037,7 +5037,7 @@
 		<value>Type de paiement</value>
 	</data>
 	<data name="R_RENEWALUPLOAD" xml:space="preserve">
-		<value>Renewal Upload</value>
+		<value></value>
 	</data>
 	<data name="L_EXTR_IMPORTEXTRACT1" xml:space="preserve">
 		<value></value>
```
Comment (hirensoni913) [inline IMIS/App_GlobalResources/Resource.fr-CM.resx:5043]: Translations are missing

File: IMIS/App_GlobalResources/Resource.fr-cd.resx [hunk 3]
```diff
@@ -5034,10 +5034,10 @@
 		<value>Tous les détails</value>
 	</data>
 	<data name="L_PAYMENTTYPE1" xml:space="preserve">
-		<value>Payment Type</value>
+		<value></value>
```
Comment (hirensoni913) [inline IMIS/App_GlobalResources/Resource.fr-cd.resx:5037]: Missing translations

File: IMIS/App_GlobalResources/Resource.fr-dj.resx [hunk 4]
```diff
@@ -5037,7 +5037,7 @@
 		<value>Type de paiement</value>
 	</data>
 	<data name="R_RENEWALUPLOAD" xml:space="preserve">
-		<value>Renewal Upload</value>
+		<value></value>
```
Comment (hirensoni913) [inline IMIS/App_GlobalResources/Resource.fr-dj.resx:5040]: Missing Translations

File: IMIS/App_GlobalResources/Resource.my.resx [hunk 5]
```diff
@@ -4887,157 +4887,157 @@
 		<value>ဖော်ပြရန်မက်ဆေ့ခ်ျမရှိပါ</value>
 	</data>
 	<data name="L_GENDER1" xml:space="preserve">
-		<value>Gender</value>
+		<value></value>
```
Comment (hirensoni913) [inline IMIS/App_GlobalResources/Resource.my.resx:4890]: Missing Translations

File: IMIS/Home.aspx [hunk 6]
```diff
@@ -70,7 +70,7 @@ In case of dispute arising out or in relation to the use of the program, it is s
 
 
 <asp:Content ID="Content1" ContentPlaceHolderID="Body" runat="Server">
-    <span id="version">v1.6.2 (build <asp:Label ID="compiledVersion" runat="server" Text=''></asp:Label>)
+    <span id="version">v1.5.2 (build <asp:Label ID="compiledVersion" runat="server" Text=''></asp:Label>)
```
Comment (hirensoni913) [inline IMIS/Home.aspx:73]: Why is it changed from 1.6.2 to 1.5.2?

**Pull request closed without merging**